experiment/simulation/index.html

```diff
@@ -1,43 +1,185 @@
+                                </br>
+                                </br>
+
+                                <input class="input " style="width: 100px; float:left; margin: 0.25rem;" id="x1" type="number" placeholder="X" step=".01" value="0">
+                                <input class="input " style="width: 100px; float:left;margin: 0.25rem;" id="y1" type="number" placeholder="Y" step=".01" value="0">
+                                <input class="input " style="width: 100px; float:left;margin: 0.25rem;" id="z1" type="number" placeholder="Z" step=".01" value="0">
+                                <div class="control">
+                                    <button class="buttonissecondary" id="modalBox_addsubmit" style="width: 10rem; float:left; margin: 1rem;">Add!</button>
+                                </div>
+                            </p>
+                        </div>
+
+                    </div>
+
+                </p>
+
+                <div class="v-datalist-title" id="top-border">Point Co-ordinates</div>
+
+                <div style="display: flexbox;">
+                    X:
+                    <input class="input is-responsive is-small" style="width:20%" type="number" id="quantityx" value="3" placeholder="X"> Y:
+                    <input class="input is-responsive is-small" style="width:20%" type="number" id="quantityy" value="3" placeholder="Y"> Z:
+                    <input class="input is-responsive is-small" style="width:20%" type="number" id="quantityz" value="3" placeholder="Z">
+                </div>
+                <input class="button is-responsive is-small is-link" style="margin: 2px;" id="move-button" type="submit" value="Move">
+
+                <div class="v-datalist-title" id="top-border">World and Instances</div>
+
+                <div>
+                    <div style="margin: 0.2rem;">
+                        <label for="quantity">X:</label>
+                        <input class="input is-responsive is-small" style="width:20%" type="number" id="h-x" value="3" placeholder="X">
+                        <label for="quantity">Y:</label>
+                        <input class="input is-responsive is-small" style="width:20%" type="number" id="h-y" value="3" placeholder="Y">
+                        <label for="quantity">Z:</label>
+                        <input class="input is-responsive is-small" style="width:20%" type="number" id="h-z" value="3" placeholder="Z">
+                    </div>
+
+                    <div style="margin: 0.2rem;">
+                        <label for="quantity" style="margin-top: 0.2rem;">Scaling Factor</label>
+                        <input class="input is-responsive is-small" style="width:20%; margin: 0.2rem;" type="number" id="h-s" value="1" step="0.01" placeholder="Scale">
+                    </div>
+                </div>
+
+
+                <div class="v-datalist-row v-bold" id="output1"></div>
+                <div class="v-datalist-row v-bold" id="output2"></div>
```

Comment:
Can we have more meaningful names for output1 and output2. I am not sure if 1 and 2 carry any meaning in the identifier

Reply:
removed them sir, since they are no longer needed